pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -138,6 +154,26 @@ func (h *NetworkUtilsHandler) IptablesAppendRule(table, chain string, rulespec .
 
 	return iptablesObject.Append(table, chain, rulespec...)
 }
+func (h *NetworkUtilsHandler) NftablesNewChain(table, chain string) error {
```

Comment:
this function always returns nil, even if the backend will fail to add the chain. is it really your intention? I think we should return nil only when we are sure that the chain is present (either b/c it existed before we called the function, or b/c we created it). Otherwise we should know that an error occurred.

Reply:
nice catch!